glitch/parsers/cmof.py

```diff
@@ -1497,19 +1504,20 @@ def process_list(name, value, start_line, end_line):
                     k_values.append(k)    
             elif isinstance(keyvalue, list) and type == "attribute":
             # block (ex: access {} or dynamic setting {}; blocks of attributes; not allowed inside local values (variables))
-                if name == "dynamic":
-                    for block in keyvalue:
-                        for block_name, block_attributes in block.items():
-                            k = create_keyvalue(block_attributes["__start_line__"], 
-                                    block_attributes["__end_line__"], f"dynamic.{block_name}", None)
-                            k.keyvalues = self.parse_keyvalues(unit_block, block_attributes, code, type)
-                            k_values.append(k)
-                else:
+                try:
                     for block_attributes in keyvalue:
                         k = create_keyvalue(block_attributes["__start_line__"], 
                                 block_attributes["__end_line__"], name, None)
                         k.keyvalues = self.parse_keyvalues(unit_block, block_attributes, code, type)
                         k_values.append(k)
+                except Exception:
```

Comment:
What is the exception that can happen here?

Reply:
I am not sure, but i remember that it was related to the dynamic "flag" that we can have for a block of attributes. For example: if we have: 
`dynamic block_name {
     at1 = val1
     }`

Reply:
I think you we can simply remove the "Exception" word. If body of try fails, then executes the except body. I tested and I do not know why did I put "except Exception" instead of only "except". If you would like to try in your machine, you can create a .tf script with the following code: ` and remove the except block so that you can check the error.

Reply:
Ok thanks João! I'll try that